src/chart/options/options.cpp

```diff
@@ -22,6 +22,10 @@ void Options::reset()
 	channels.reset();
 	title = title.get().has_value() ? Title(std::string())
 	                                : Title(std::nullopt);
+	subtitle = subtitle.get().has_value() ? Title(std::string())
```

Comment:
We should rename Title type to something more general. if we use it for the footer, subtitle.

Reply:
ok